Assets/UI/WorldView/citybannermanager_CQUI.lua

```diff
@@ -956,7 +962,6 @@ function Initialize_CQUI()
     LuaEvents.CQUI_SettingsInitialized.Add(CQUI_OnSettingsInitialized);
     LuaEvents.CQUI_SettingsUpdate.Add(CQUI_OnSettingsUpdate);
 
-    Events.CityRemovedFromMap.Add(CQUI_OnCityRemovedFromMap);    -- CQUI erase real housing from improvements data everywhere when a city removed from map
```

Comment:
Deleting this line here does not seem related to the the change for this PR. What is the reason for this deletion?

Reply:
Disregard this comment. I saw that it's related to some other bug fix that's already been approved and merged to trunk.